src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -39,17 +39,27 @@ public static final String getHelpSummary() {
                 .append("GENERAL COMMANDS\n")
```

Comment:
Was actually thinking to let @wengfaing do this part since he needs some boost to meet his functional code req! (Maybe for the other help start window)

Reply:
Or maybe can ask @wengfaing to make a separate message to display for help on start up and then later change it to show that instead

Reply:
Nah its ok! whats done is done dont have to revert back because you alrd spent the time and effort